setup.py

```diff
@@ -111,6 +111,7 @@
         'piradon',
         'matplotlib',
         'h5py',
+        'importlib-metadata>=4.8;python_version<"3.8"',

```

Comment:
Will this prevent the install for python versions above 3.8? If yes, then we need to find another solution as many students will just use whatever python they get by default (typically 3.9 or 3.10 at the moment).

Reply:
No, it's a condition. So it will only install for python before 3.8. It's a backport of the Python 3.8 standard library `importlib.metadata`. One of the dependencies is missing this package in its own requirements.txt file. I didn't bother to work out which. Safer for students on Python 3.7 or less to just have it installed.